### PR TITLE
Adds Slaught-o-Matic disk

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
Adds the Slaught-o-Matic to disk form, magazine included, perfect for arming your own vagabond army! 

## Why It's Good For The Game
adds a new, cheap plastic people popper

## Changelog
:cl:
add: Added Slaught-o-Matic disk
add: Slaught-o-Matic disk now in disk vendor
spellcheck: fixed a few typos in code
/:cl:

### CODE REVIEW

**Comment by AgentMA1** [comment]
>barges in
>"Cringe"
>Leaves

**Comment by Sparrowbird611** [comment]
Why does it cost 1500 to make 6 slaught-o-matics when it would cost 600 to just buy them (last I checked they were 100 per SMG). If you're gonna do this it shouldn't be a scam.

Why does this even exist anyway, there's like 50 slaught-o-matics in the vendor as is, who is gonna buy this disk? Honestly, if you want to make this, I'd make it cheaper than it would be to buy this many slaught-o-matics, maybe 400-500, since you also need to add supplies, print the guns, and oldify the guns in the printer. I think it'd be kind of pointless still, but at least worth more than it is rn.

**Comment by zombiemimic** [comment]
> Why does it cost 1500 to make 6 slaught-o-matics when it would cost 600 to just buy them (last I checked they were 100 per SMG). If you're gonna do this it shouldn't be a scam.
> 
> Why does this even exist anyway, there's like 50 slaught-o-matics in the vendor as is, who is gonna buy this disk? Honestly, if you want to make this, I'd make it cheaper than it would be to buy this many slaught-o-matics, maybe 400-500, since you also need to add supplies, print the guns, and oldify the guns in the printer. I think it'd be kind of pointless still, but at least worth more than it is rn.

I was just going to do that, actually

File: code/game/objects/items/weapons/design_disks/frozen_star.dm [hunk 1]
```diff
@@ -321,6 +321,14 @@
 		/datum/design/autolathe/ammo/smg/rubber,
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/fs_slaught_o_matic
+	disk_name = "Frozen Star - .35 Slaught-o-Matic HG"
+	icon_state = "frozenstar"
+	rarity_value = 7
+	license = 12
+	designs = list(
+		/datum/design/autolathe/gun/slaught_o_matic = 1, // "FS HG .35 \"Slaught-o-Matic\""
```
Comment (Firefox13) [inline code/game/objects/items/weapons/design_disks/frozen_star.dm:330]:
```suggestion
		/datum/design/autolathe/gun/slaught_o_matic = 1 // "FS HG .35 \"Slaught-o-Matic\""
```

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**